x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/crawler_overview.tsx

```diff
@@ -36,22 +40,94 @@ export const CrawlerOverview: React.FC = () => {
+              <EuiTitle size="s">
+                <h3>
+                  {i18n.translate('xpack.enterpriseSearch.appSearch.crawler.domainsTitle', {
+                    defaultMessage: 'Domains',
+                  })}
+                </h3>
+              </EuiTitle>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <AddDomainFlyout />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+          <EuiSpacer size="m" />
+          <DomainsTable />
+        </>
+      ) : (
+        <>
           <EuiTitle size="s">
             <h3>
-              {i18n.translate('xpack.enterpriseSearch.appSearch.crawler.domainsTitle', {
-                defaultMessage: 'Domains',
+              {i18n.translate('xpack.enterpriseSearch.appSearch.crawler.empty.title', {
+                defaultMessage: 'Add a domain to get started',
               })}
             </h3>
           </EuiTitle>
-        </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          <AddDomainFlyout />
-        </EuiFlexItem>
-      </EuiFlexGroup>
-      <EuiSpacer size="m" />
-      <DomainsTable />
+          <EuiText>
+            <p>
+              {i18n.translate('xpack.enterpriseSearch.appSearch.crawler.empty.description', {
+                defaultMessage:
+                  "Easily index your website's content. To get started, enter your domain name, provide optional entry points and crawl rules, and we will handle the rest.",
+              })}{' '}
+              <EuiLink external target="_blank" href={`${DOCS_PREFIX}/web-crawler.html`}>
+                {i18n.translate(
+                  'xpack.enterpriseSearch.appSearch.crawler.empty.crawlerDocumentationLinkDescription',
+                  {
+                    defaultMessage: 'Learn more about the web crawler',
+                  }
+                )}
+              </EuiLink>
+            </p>
+          </EuiText>
+          <AddDomainForm />
+          <EuiSpacer />
+          <EuiFlexGroup justifyContent="flexEnd">
+            <EuiFlexItem grow={false}>
+              <AddDomainFormSubmitButton />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </>
+      )}
+      {(crawlRequests.length > 0 || domains.length > 0) && (
+        <>
+          <EuiSpacer size="xl" />
+          <EuiTitle size="s">
+            <h3>
+              {i18n.translate('xpack.enterpriseSearch.appSearch.crawler.crawlRequestsTitle', {
+                defaultMessage: 'Recent Crawl Requests',
+              })}
+            </h3>
+          </EuiTitle>
+          <EuiSpacer size="xs" />
+          <EuiText color="subdued" size="s">
+            <p>
+              {i18n.translate('xpack.enterpriseSearch.appSearch.crawler.crawlRequestsDescription', {
+                defaultMessage:
+                  "Recent crawl requests are logged here. Using the request ID of each crawl, you can track progress and examine crawl events in Kibana's Discover or Logs user interfaces. ",
```

Comment:
space nit at end of copy
```suggestion
                  "Recent crawl requests are logged here. Using the request ID of each crawl, you can track progress and examine crawl events in Kibana's Discover or Logs user interfaces.",
```